Makefile

```diff
@@ -37,6 +37,18 @@ help: ## Display this help.
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
+.PHONY: fmt
```

Comment:
You may want to consider using the Makefile contents from https://github.com/nephio-project/nephio/blob/main/Makefile

Reply:
added the new makefile